pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "Keystores":
+
+`gopass mounts`
+
+- Mount a shared git "keystore":
+
+`gopass mounts add keystore_name location_git_repo`
+
+- Search interactive using a keyword:
```

Comment:
Shouldn't it be "Search interactively..."? as "Search interactive..." sounds a little odd.

Reply:
:+1: